node/subsystem/src/messages.rs

```diff
@@ -272,6 +293,8 @@ pub enum RuntimeApiRequest {
 	ValidationCode(ParaId, BlockNumber, Option<BlockNumber>, oneshot::Sender<ValidationCode>),
 	/// Get head data for a specific para.
 	HeadData(ParaId, oneshot::Sender<HeadData>),
+	/// Get a the candidate pending availability for a particular parachain by parachain / core index
+	CandidatePendingAvailability(ParaId, oneshot::Sender<Option<CommittedCandidateReceipt>>),
```

Comment:
I couldn't see any way around creating this runtime request, but I could have missed something.

Reply:
Yeah, this not existing is a byproduct of #1419 not being done.